nautobot/core/templates/buttons/add.html

```diff
@@ -1,5 +1,13 @@
 {% if add_url %}
-    <a href="{{ add_url }}" class="btn btn-primary" id="add-button">
-        <span class="mdi mdi-plus-thick" aria-hidden="true"></span> Add
-    </a>
+    {% if list_element %}
+        <li>
+            <a href="{{ add_url }}" id="add-button">
+                <span class="mdi mdi-plus-thick text-muted" aria-hidden="true"></span> Add
```

Comment:
I wonder - should we change this to `Add <verbose_name>` for more clarity?

Reply:
We talked about doing this for the Edit button on the Detail view, we could do the same here.

Reply:
Is this needed? We aren't currently passing context. I could pass in context, but just wanted clarity.